consensus/scheme/rolldpos/rolldposctx.go

```diff
@@ -458,7 +458,9 @@ func (ctx *rollDPoSCtx) Commit(msg interface{}) (bool, error) {
 	if pendingBlock == nil {
 		return false, nil
 	}
-	ctx.logger().Info("consensus reached", zap.Uint64("blockHeight", ctx.round.Height()))
+	if ctx.round.Height()%100 == 0 {
```

Comment:
can we do DEBUG on this one w/o sampling?

Reply:
i still want to keep this to indicate consensus OK, similar to `committed a block` above

it only adds 1 line every 100 blocks (8 minutes), so won't increase log size